modules/dhp_connected_devices/app/controllers/dhp_connected_devices/fitbit/fitbit_controller.rb

```diff
@@ -63,6 +64,20 @@ def feature_enabled
         routing_error unless Flipper.enabled?(:dhp_connected_devices_fitbit, @current_user)
       end
 
+      def user_verified
+        connection_unavailable_error if @current_user&.icn.blank?
+      end
+
+      def connection_unavailable_error
+        Rails.logger.warn('Device connection unavailable for Veterans without an ICN')
+        error = Common::Exceptions::Forbidden.new(
+          detail: 'User with an invalid ICN value attempted to connect their Fitbit',
+          source: 'FitbitController'
+        )
+        log_message_to_sentry('User with an invalid ICN value attempted to connect their Fitbit', 'warn')
+        raise error
```

Comment:
That's my bad  because I made these changes 😂
But can we simplify this a bit by moving line 77 (log to sentry) to right after the Rails.logger. Then just raise the exception inline on the next line?

Reply:
updated